proto/ibc/core/module/v1/module.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package ibc.core.module.v1;
+
+import "cosmos/app/v1alpha1/module.proto";
+
+// Module is the config object of the core module.
+message Module {
+  option (cosmos.app.v1alpha1.module) = {
+    go_import: "github.com/cosmos/ibc-go/modules/core"
+  };
+
+  // authority defines the custom module authority. If not set, defaults to the governance module.
+  string authority = 1;
```

Comment:
any idea why this is on the proto definition? and where it gets set?

Reply:
In `appmodule.Register` we pass an empty `&modulev1.Module{}` - same for sdk modules maintained over there afaik